playwright/test/view-items.test.ts

```diff
@@ -236,19 +222,12 @@ test.skip('(15) | The location of the search results should be displayed', async
   );
   await itemWithSearchAndStructuresAndQuery(context, page);
 
-  if (isMobile(page)) {
-    await page.getByRole('button', { name: 'Show info' }).click();
-  }
+  await accessSidebarOnMobile(page);
 
   await page
```

Comment:
should we change this to and expect().toBeVisible()?

Reply:
absolutely